cfme/generic_objects/instance/rest.py

```diff
@@ -44,6 +45,39 @@ def create(self, name, definition, attributes=None, associations=None):
     return entity
 
 
+@MiqImplementationContext.external_for(GenericObjectInstanceCollection.all, ViaREST)
+def all(self):
```

Comment:
totally optional:
I would try to compress this stuff... like
```
In [1]: generic_objs = []                                                                                                                          

In [2]: for gen_obj  in appliance.rest_api.collections.generic_objects: 
   ...:     gen_def_rest = appliance.rest_api.collections.generic_object_definitions.find_by(id=gen_obj.generic_object_definition_id)[0] 
   ...:     gen_def = appliance.collections.generic_object_definitions.instantiate(name=gen_def_rest.name, description=gen_def_rest.description) 
   ...:     generic_objs.append(appliance.collections.generic_objects.instantiate(name=gen_obj.name, definition=gen_def)) 
   ...:                                                                                                                                            
(rest-api) [RESTAPI] GET https://192.168.122.231/api/generic_objects {'expand': 'resources'}
(rest-api) [RESTAPI] GET https://192.168.122.231/api/generic_object_definitions {'filter[]': ['id = "15"']}
(rest-api) [RESTAPI] GET https://192.168.122.231/api/generic_object_definitions/15 {}

In [3]: generic_objs                                                                                                                               
Out[3]: [GenericObjectInstance(name='generic_instance_003', definition=GenericObjectDefinition(name='generic_class_485', description='Generic Object Definition', attributes=None, associations=None, methods=None, custom_image_file_path=None, rest_response=None), attributes=None, associations=None, rest_response=None, my_service=None)]
```

Reply:
Oh good point, let me see what I can do

Reply:
Done, please take a look @digitronik